lib/internal/bootstrap/pre_execution.js

```diff
@@ -1,18 +1,51 @@
 'use strict';
 
 const {
+  Object,
   Map,
   ObjectDefineProperty,
   SafeWeakMap,
 } = primordials;
 
 const { getOptionValue } = require('internal/options');
 const { Buffer } = require('buffer');
-const { ERR_MANIFEST_ASSERT_INTEGRITY } = require('internal/errors').codes;
+const {
+  ERR_MANIFEST_ASSERT_INTEGRITY,
+  ERR_INVALID_OPT_VALUE,
+  ERR_PROTO_ACCESS,
+} = require('internal/errors').codes;
 const assert = require('internal/assert');
 
+function patchObjectProto() {
+  const disableProto = getOptionValue('--disable-proto');
+  switch (disableProto) {
+    case '':
+      // Flag was not passed
+      break;
+    case 'delete':
+      // eslint-disable-next-line no-proto
+      delete Object.prototype.__proto__;
```

Comment:
Is it possible to add some logging for this operation?

Reply:
I'm not sure what that means

Reply:
I mean, write to debuglog some message that __proto__ field successfully removed from prototype. But now I’ve thought a little and consider it useless.